.travis.yml

```diff
@@ -58,9 +58,6 @@ matrix:
     - language: generic
```

Comment:
Within the `before_install` section can we get a `python -c "import ssl; print(ssl.OPENSSL_VERSION)"`

Reply:
In the `before_install`, this would show the default Python version, so I added it in _travis/install.sh instead. What do you think?

Reply:
I think it's exactly what I actually wanted :) Thanks.